### DISCUSSION TITLE
Image Path when the file isn't in the root

### DISCUSSION THREAD

**Discussion by PeterGyger** [community/community#23413] [Other Feature Feedback, Questions, & Ideas]
<p>Hello</p>
<p>When i have my image in a folder named “Image” and my file “sample.md” is in the Root, then i access the picture “pix.jpg” with the path “image\pix.jpg”. But how i access the the picture, when the file “sample.md” is in a subfolder?</p>
<p>That means “sample.md” ist in the folder “/_posts” and the pictures are in the folder “/image”. I tryed with the prefix “[repository]/image/pix.jpg”. Nothing works. Please a little information to solve my confusion.</p>
<p>Greetings<br>
Peter</p>

**Comment by ghost** [comment]
<p>Hey there <a class="mention" href="/petergyger">@petergyger</a>, help’s arrived. </p>
<p>You can reference static files (like images) to your Jekyll posts by adding it them to a subdirectory like <strong>/assets</strong> and then referencing that in Markdown: <strong><img src="/assets/screenshot.png" alt="a screenshot example"></strong>. There’s some more info on this over here: <a href="https://jekyllrb.com/docs/posts/#including-images-and-resources" rel="nofollow noopener">https://jekyllrb.com/docs/posts/#including-images-and-resources</a> 🙂</p>
<p>Let me know how you go</p>

**Comment by PeterGyger** [comment]
<p>Hello</p>
<p>So sorry - but don’t work. the Directory Structure:</p>
<p>[Repro Test]</p>
<p>¦-_posts</p>
<p>¦-images</p>
<p>In the directory “_posts” is a markdown file “sample.md”. In this file i link the Image “pic.jpg” from the Folder “images”. How? There is no way that work. I habe no Problems, when  the Markdown File is in the Root and i write a static link to “images/pic.jpg”.</p>
<p>I read the Jeckyll Manual and i followed your link. But for my question there is no solution.</p>
<p>Greetings</p>
<p>Peter</p>

**Comment by ghost** [comment, marked as answer]
<p>Thanks for letting me know <a class="mention" href="/petergyger">@petergyger</a>,</p>
<p>Let’s check if adding a forward slash before the reference helps. For example:</p>
<pre><code>![this screenshot](/assets/pic.png)
</code></pre>
<p>If that doesn’t help, let’s check if adding " <strong>…</strong>" to before the first forward slash helps:</p>
<pre><code>![this screenshot](../assets/pic.png)
</code></pre>

**Comment by Wabri** [comment]
<p>The path of the link in github markdown follow the linux convention, so the second example that give <a class="mention" href="/timrossback">@timrossback</a> should work. If not you can give to us the tree of the projects/repository and we can help in a better way.</p>
<p>There is another way to link image, using the absolute path like this:</p>
<pre><code>![this screenshot](https://github.com/username/repository/blob/branch/path/pic.png)
</code></pre>
<p>-Gabriele-</p>

**Comment by PeterGyger** [comment]
<p>Hello</p>
<p>You win  :slight_smile:</p>
<p>I use the two dots and the Problem was solved.</p>
<p>Thank you very match for your kind help</p>
<p>Greetings</p>
<p>Peter</p>

**Comment by ghost** [comment]
<p>You’re more than welcome <a class="mention" href="/petergyger">@petergyger</a>! </p>
<p>If you need a hand with anything else, just let me know or feel free to post a new thread in the Conversations board 🌟</p>
<p>Have a great day!</p>

**Comment by aartirajput1** [comment]
<p>Hi All</p>
<p>I am also facing the issue for relative link. following the same structre as above , only diffence is i need to link .md file from _pages/ to _posts/.md.</p>
<p>i can see .md file <a href="https://aartirajput1.github.io/JekyllSiteRepo/JekyllSiteRepo/adding/" rel="nofollow noopener">https://aartirajput1.github.io/JekyllSiteRepo/JekyllSiteRepo/adding/</a> but not rendedered in HTML , its showing 404 error not found.</p>
<p>_post/.md file : <a href="https://github.com/aartirajput1/JekyllSiteRepo/blob/master/_posts/2019-04-05-Homepage.markdown" rel="nofollow noopener">https://github.com/aartirajput1/JekyllSiteRepo/blob/master/_posts/2019-04-05-Homepage.markdown</a></p>
<p>Please help.</p>
<p>Thanks in advance :slight_smile:</p>

**Comment by Wulfheart** [comment]
<p>Same issue here.</p>

**Comment by aayushhh411** [comment]
<p>Here’s one method I use frequently. Whenever I need to fetch an image from somewhere. I don’t save it in the root folder. Instead what I do is I try to upload it somewhere and take the image address from there to fetch directly. The one service I use is the google blogger. Where I’ve created a blog, where I create draft posts for saving images online. And by right-clicking, I can get the image address. I think this might be helpful for you.</p>
<p>Source: <a href="https://exactlyhowlong.com/" rel="noopener nofollow ugc">ExactlyHowLong</a></p>

**Comment by sayedabdallah** [comment]
I have the same issue . although Images appears locally using VS Code when I push my code to github the images is not showing

my folder structure is

-images
-----image.png
-Notebooks
-----FirstNotebook.md  # here i use ![my image] ( ../images/image.png) I just put extra space between [] and () here to show the code
